vm/src/slots.rs

```diff
@@ -4,14 +4,15 @@ use crate::VirtualMachine;
 
 bitflags! {
     pub struct PyTpFlags: u64 {
+        const HEAPTYPE = 1 << 9;
         const BASETYPE = 1 << 10;
         const HAS_DICT = 1 << 40;
     }
 }
 
 impl PyTpFlags {
     // CPython default: Py_TPFLAGS_HAVE_STACKLESS_EXTENSION | Py_TPFLAGS_HAVE_VERSION_TAG
-    pub const DEFAULT: Self = Self::from_bits_truncate(0);
+    pub const DEFAULT: Self = Self::HEAPTYPE;
```

Comment:
This is not a default flag of CPython by the upper comment. Can this value be injected when we are actually creating a heap type? Similar as `HAS_DICT` flag?

Reply:
The thing is, we don't actually have non-heap types yet; all of our type objects are allocated on the heap.